media-libs/libcamera/libcamera-0.6.0.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit meson
+
+DESCRIPTION="A complex camera support library for Linux, Android, and ChromeOS"
+HOMEPAGE="https://libcamera.org"
+SRC_URI="https://gitlab.freedesktop.org/camera/libcamera/-/archive/v${PV}/libcamera-v${PV}.tar.bz2 -> ${P}.tar.bz2"
+S="${WORKDIR}/libcamera-v${PV}"
+LICENSE="Apache-2.0 CC0-1.0 BSD-2 CC-BY-4.0 CC-BY-SA-4.0 GPL-2+ GPL-2 LGPL-2.1+"
+SLOT="0"
+KEYWORDS="~arm64"
+IUSE="drm gnutls openssl gstreamer jpeg tiff libevent qt6 sdl trace udev unwind v4l"
```

Comment:
The ebuild must integrate the testsuite. This is useful for a lib, even if we can launch only a few in a build env.

Reply:
I didn't checked exactly but please verify if a testsuite can be launched during src_test.
If an option is required, use the test useflag for it.